streams/writable-streams/properties.js

```diff
@@ -0,0 +1,204 @@
+'use strict';
+
+if (self.importScripts) {
+  self.importScripts('/resources/testharness.js');
+}
+
+// The purpose of this file is to test for objects, attributes and arguments that should not exist.
+// The test cases are generated from data tables to reduce duplication.
+// TODO(ricea): This file can be deleted when automatic IDL checking has advanced to cover these.
```

Comment:
I didn't realise that we have aspects that cannot be expressed in IDL when I wrote it. Removed.